src/components/Common/Portfolio.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import PortfolioItem from './PortfolioItem';
+
+
+import img1 from '../assets/img/portfolio/dreams-preview.png';
```

Comment:
You should be more explicit on you variables name. `img1` say nor more than it's an image